app/helpers/gcse_qualification_helper.rb

```diff
@@ -17,6 +17,19 @@ def heading_for_gcse_edit_details(subject)
     t("gcse_edit_details.heading.#{subject}")
   end
 
+  def guidance_for_gcse_edit_details(subject, qualification_type)
+    if I18n.exists?("gcse_edit_details.guidance.#{subject}.#{qualification_type}")
+      tag.p(t("gcse_edit_details.guidance.#{subject}.#{qualification_type}"), class: 'govuk-body')
```

Comment:
Nit: Seeing `tag.p` feels like a code smell, should this be in the view?

Reply:
As the guidance (and this paragraph) only appears if there’s guidance, if this were in the view, there would need to be a lot of logic there to determine whether to output the HTML or not. As this is a helper function, seems like an okay place for this to live, bit happy to defer to those more familiar with Ruby and conventions used in this app.

Reply:
It's a view helper, they're definitely fine to generate HTML 👍